docs/devGuide/settingUp.md

```diff
@@ -10,39 +10,58 @@
 
 <div class="lead">
 
-This page explains how to set up your development environment to start contributing to MarkBind.
+This page explains how to set up your development environment to start
+contributing to MarkBind.
+
 </div>
 
 ## Prerequisites
 
-1. **Node.js** ({{ node_version }} or higher) with<br>
-   **npm** v7 or higher
-1. **Java** 8 or later, and<br>
-   **Graphviz** v2.38 or later<br>
-   %%(The above two are required for one of the third-party library used by MarkBind)%%
-1. **Verify** that all tools are accessible and the versions are as expected by running these commands in the console.
-   * `node --version`
-   * `npm --version`
-   * `java --version`
-   * `dot -V` (for Graphviz)
+1. **Node.js** ({{ node_version }} or higher) with<br> **npm** v7 or higher
+1. **Java** 8 or later, and<br> **Graphviz** v2.38 or later<br> %%(The above two
+   are required for one of the third-party library used by MarkBind)%%
+1. **Verify** that all tools are accessible and the versions are as expected by
+   running these commands in the console.
+   - `node --version`
+   - `npm --version`
+   - `java --version`
+   - `dot -V` (for Graphviz)
 
 <box type="tip" seamless>
 
-We recommend the **WebStorm IDE** for working with MarkBind code.
-</box>
+We recommend the **WebStorm IDE** for working with MarkBind code. </box>
```

Comment:
Unnecessary change here

Reply:
I think I may have accidentally formatted the document here oops